src/modules/control.cpp

```diff
@@ -34,13 +34,70 @@ static uint16_t targetVolume;
 static uint32_t totalBreathTime;
 static uint32_t targetInhalationTime;
 
+// Added
+static uint32_t targetExhalationTime;
```

Comment:
Point of discuss we need to have it whether or not we need to control exhalation. So I would say this is somewhat of an open

Reply:
No problem. We can add/remove it as necessary.